packages/peregrine/lib/talons/CheckoutPage/useCheckoutPage.js

```diff
@@ -163,9 +164,10 @@ export const useCheckoutPage = props => {
                     }
                 });
 
+                // Cleanup stale cart and customer info.
                 await removeCart();
-
                 await clearCartDataFromCache(apolloClient);
+                await clearCustomerDataFromCache(apolloClient);
```

Comment:
Was a bit surprised to see this - what customer data needs cleaned up after checkout? Wouldn't it remain the same?

Reply:
Hmm maybe this should be removed - I thought it was necessary but if we have customer data it's only for a logged in user and we don't need to clear that on checkout.